### PR TITLE
Migrate resource tree for resource token to react

### PR DESCRIPTION
**Pull request by victor-meng**
[Preview this branch](https://cosmos-explorer-preview.azurewebsites.net/pull/956?feature.someFeatureFlagYouMightNeed=true)

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "GITHUB_CLIENT_ID": "167ea4b09801db1de03d",
+  "GITHUB_CLIENT_SECRET": "e1208e29f4e03c5f9c6fc08a36375df8344ab01d"
+}
```
Comment (southpolesteve) [inline config.json:4]: Did you mean to check this in?

File: less/tree.less [hunk 2]
```diff
@@ -1,272 +1,271 @@
 @import "./Common/Constants";
 
-
 .resourceTree {
+  height: 100%;
+  flex: 0 0 auto;
+  width: 20%;
```
Comment (southpolesteve) [inline less/tree.less:6]: Verify that the collapsing the resource tree still works. I know this is something we broke earlier. The tabs wouldn't fill the space.

File: config.json (hunk 1 above)
Comment (victor-meng) [reply config.json:4]: Nop! This is the config for testing github connection. I'll remove it. Thanks for catching this!

**Review by southpolesteve — APPROVED** [review]

File: less/tree.less (hunk 2 above)
Comment (victor-meng) [reply less/tree.less]: This will indeed break collapsing the resource tree. I've reverted the change.

**Pull request merged**